### PR TITLE
Sprint 01

### PR DESCRIPTION
**Pull request by Usmanathunga**
Completed sprint 1

### CODE REVIEW

File: ramp-up-web/public/index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="https://kendo.cdn.telerik.com/themes/5.6.0/default/default-main.css"></link>
+        <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <style>
```
Comment (shamal-iroshan) [inline ramp-up-web/public/index.html:6]: Don't put styles in the index.html file make a separate stylesheet and attach it to the index.js file

**Review by shamal-iroshan — CHANGES_REQUESTED** [review]
fix the commented issue

File: ramp-up-web/public/index.html (hunk 1 above)
Comment (Usmanathunga) [reply ramp-up-web/public/index.html:6]: Ok I will fix this issue!!

**Review by shamal-iroshan — APPROVED** [review]

**Comment by Usmanathunga** [comment]
Age calculated
-Sprint 2-
Initialize Backend Project
Database Connectivity

**Pull request merged**